fecgov/fecfile-web-api
# feature/1978 - independent expenditure presidential primary candidate state not saving

[#1978](https://github.com/fecgov/fecfile-web-app/issues/1978)
  
The issue was that the candidate state was not saving on an independent expenditure with presidential primary.  This was by design as the candidate contact should not be updated for presidential primaries.  The solution was to add a candidate state field to the schedule e transaction and use that in this case.

## How to test

Add independent expenditure with candidate contact state set for a presidential primary.  Confirm this state appears when transaction is loaded.
